api/capi/src/nnstreamer-capi-util.c

```diff
@@ -1140,6 +1141,17 @@ ml_validate_model_file (char **model, unsigned int num_models,
       if (g_ascii_strcasecmp (file_ext[0], ".dlc") != 0) {
         status = ML_ERROR_INVALID_PARAMETER;
       }
+#endif
+      break;
+    case ML_NNFW_TYPE_PYTORCH:
+#if !defined (__ANDROID__)
```

Comment:
need test.
I think this condition (non-android) is unnecessary.
If sub-plugin pytorch is not installed on the target, nnfw-availability will return the error 'not-supported'.

Reply:
Updated!